week2/homework/src/index.js

```diff
@@ -1,3 +1,115 @@
 'use strict';
+let fs = require('fs');
```

Comment:
When there is no re-assignment of the variable, you should use `const`

Reply:
Thanks for your feedback! I will update it! 👍